application/src/main/java/org/togetherjava/tjbot/commands/code/CodeMessageAutoDetection.java

```diff
@@ -38,11 +40,14 @@ public CodeMessageAutoDetection(Config config, CodeMessageHandler codeMessageHan
 
         isHelpForumName =
                 Pattern.compile(config.getHelpSystem().getHelpForumPattern()).asMatchPredicate();
+
+        ignoredRolePattern = config.getIgnoreMessageAutoDetectionRolePattern();
```

Comment:
I will refactor it to Pattern.compile... please note that I had Pattern.compile.. at the beginning of the pull request and was asked by taz to switch to String. Please also consult with him again.

Reply:
Ah, I see what was the issue.

`List<Predicate<String>> isIgnoreCodeActionsRole;` no need for a list here, just `Predicate<String>`, since it's just one regex.

Reply:
done sir!